.travis.yml

```diff
@@ -0,0 +1,50 @@
+sudo: false
+language: python
+python:
+  - 3.4
+  - 3.5
+  - 3.6
+
+cache:
+  pip: true
+
+before_install:
+  - if [[ $TRAVIS_OS_NAME == osx ]]; then TRAVIS_PYTHON_VERSION=3.5; fi
+
+  - >
+    if [[ $TRAVIS_OS_NAME == osx ]]; then
```

Comment:
This isnt being used.
If you want an OSX build, you need to add it to the build matrix.